conversion_sr14_sr15/README.md

```diff
@@ -0,0 +1,74 @@
+|emp_total, all employment categories| Employment categories are calculated by applying the share of an employment category at the TAZ level to the scaled SR14 employment totals, as such: <br><br><ls>1. The total employment scaling factor is calculated as follows: <br>scaling factor = sum(ABM3 emp_total) / sum(SR14 emp_total)</ls><br><br><ls> 2. SR15 emp_total = converted SR14 emp_total * scaling factor</ls><br><br><ls>3. ABM3 employment categories are aggregated to the TAZ level and the share of each employment category is calculated as: <br> ABM3_category_share = emp_category/emp_total</ls><br><br><ls> 4. The share is applied to all MGRAs in the same TAZ in the SR15 output file. SR15 emp_category = ABM3_category_share * SR15 emp_total </ls><br><br><ls> 5. Final employment values are rounded to zero decimals. </ls><br><br><ls>6. SR15 emp_total is recalculated to maintain internal consistency and correct rounding errors: SR15 emp_total = sum(all SR15 employment categories) </ls><br><br> Due to rounding errors from step 5, the sum of SR15 output emp_total does not exactly match the sum of SR15 input emp_total|
+|hs, hs_sf, hs_mf, hs_mh, enrollgradekto8, enrollgrade9to12, eollegeenroll, othercollegeenroll, hotelroomtotal|Obtained by applying MGRA crosswalk and summing together rows in the same MGRA |
+
+
+## 2.3 Consistency Checks
+Several checks were conducted to guarantee consistency across the converted synthetic population and land use files. Specifically, we checked the following in the converted SR15 land use file:
+-	Same number of MGRAs as the ABM3 land use file
+-	Taz, luz_id, and other fields were unchanged from the ABM3 land use file
+-	Sum of pop = total population records in synthetic population
+-	Sum of hh = total household records in synthetic population
+
+## 2.4 Output Files
+The conversion script outputs three files: households, persons, and land use. All three files contain the columns necessary for running the files in ABM3. Refer to the SANDAG ABM3 documentation at https://sandag.github.io/ABM/inputs.html for more information. 
+
+# 3.0 User Guide
+To run the conversion, two files are required: 
+1.	_config.yaml_ – which contains input and output directories and file names
+2.	_conversion_sr14_sr15.py_ – which contains the code to convert the files 
+
+User should clone (or download) these files to their local directory. Next, follow the steps below to convert Series 14 data to Series 15 format:  
+1.	Save the input files in the directory of your choice. Input files should include: 
+    - Series 14 synthetic person file
+    - Series 14 synthetic household file
+    - Series 14 land use file
+    - ABM3 land use file
+    - SR14 to SR15 MGRA crosswalk 
+2.	Open the config.yaml file and update the input and output directories as well as the input and output file names. 
+
+    ![image](images/config.png)
+
+3.	Open a terminal and navigate to the folder where conversion_sr14_sr15.py is saved.
```

Comment:
any specific environment needed?

Reply:
No special environment is required so long as the base anaconda3 environment is on a machine. Text has been added to line 58 in the Readme to clarify.